### PR TITLE
Add inviscid unsteady test

### PR DESCRIPTION
**Pull request by bburke38**
Adds a test for unsteady FUN3D+TACS fully coupled discipline derivatives. Right now, this only tests inviscid along the aeroelastic path. Currently, the structural functions are failing while lift and drag seem to be passing. However, changing `flow_dt` in the Python run script seems to have no impact on the test results. Changing the time step set to TACS also has limited effect.

### CODE REVIEW

**Comment by sean-engelstad** [comment]
Can you remove the .DS_store files?

File: tests/fun3d_tests/fully_coupled_disc_deriv_unsteady/euler-AE/meshes/inviscid/Adjoint/.DS_Store
Comment (sean-engelstad) [inline tests/fun3d_tests/fully_coupled_disc_deriv_unsteady/euler-AE/meshes/inviscid/Adjoint/.DS_Store:1]: @Can you delete this file and the other **/.DS_Store files?

**Review by sean-engelstad — CHANGES_REQUESTED** [review]
Please remote the DS_Store files first

File: tests/fun3d_tests/fully_coupled_disc_deriv_unsteady/euler-AE/meshes/inviscid/Adjoint/.DS_Store
Comment (bburke38) [reply tests/fun3d_tests/fully_coupled_disc_deriv_unsteady/euler-AE/meshes/inviscid/Adjoint/.DS_Store:1]: Okay, just make sure to add the necessary folders back. The .DS_Store files seem to help keep the directory structure.

**Pull request merged**